catchpoint/resource_api_test_type.go

```diff
@@ -84,22 +86,27 @@ func resourceApiTestType() *schema.Resource {
 			},
 			"start_time": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
+				Computed:    true,
 				Description: "Start time for the Test in ISO format like 2024-12-30T04:59:00Z",
 			},
 			"end_time": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
+				Computed:    true,
 				Description: "End time for the Test in ISO format like 2024-12-30T04:59:00Z",
 			},
 			"status": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "Optional. Test status: active or inactive",
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "Test status: active or inactive",
```

Comment:
The `Optional` word has been removed. Is this expected? 

Additionally, some properties which have `Optional: true` don't have the word `Optional` in the description and some of them do so checking if this should be consistent.

Pls check other places also where this is the case.

Reply:
@EmmanuelPonnudurai I have added computed true  which means this property will show changes only if user add or update the property.
 When you add computed true, there should not be default value added.
 thats the main changes done in this PR.

Reply:
Computed works with optional only.

Reply:
What I'm asking about is the Description. For example, 
 
Property `folder_id` has both `Optional: true` and `Computed: true` and `Description` starts with text `Optional`
 
Property status, has both `Optional: true` and `Computed: true`, however the `Description` _does not_ start with `Optional`